### DISCUSSION TITLE
GitHub Teams Metrics Available

### DISCUSSION THREAD

**Discussion by martedesco** [github-copilot-community/copilot-metrics-viewer#84] [Ideas]
In the latest release - v1.7 - Team metrics were made available 🎉. Take a look at the [Enterprise Team](https://docs.github.com/en/enterprise-cloud@latest/rest/copilot/copilot-usage?apiVersion=2022-11-28) and [Organization Team](https://docs.github.com/en/enterprise-cloud@latest/rest/copilot/copilot-usage?apiVersion=2022-11-28#get-a-summary-of-copilot-usage-for-a-team) API docs.

The GitHub Team is picked up at the moment the application is loaded via [an environment variable](https://github.com/github-copilot-resources/copilot-metrics-viewer?tab=readme-ov-file#vue_app_github_team) and cannot be changed in runtime. I don't think this is ideal (this is a first version due to my low availability in the past couple weeks but I wanted to make this feature available as soon as possible).

I'm thinking about having a dropdown that lists the Teams for the Organization (or Enterprise - however no API is available to list Enterprise Teams yet) and users are able to pick the Team. Once picked, the metrics across the whole application would be refreshed. Any comments on this? Where do you think the dropdown should be added (in a new tab, menu, toolbar?) Thanks

**Comment by admin-hsu1** [comment]
Hi @martedesco 

I encountered an issue with Team API, no matter what I tried, the response is always empty.
I am sure I used the right token with right permissions, as well as the correct ORG and Team Slug, can you please help provide some hints to help solve my problem?

Host ~ % curl -L \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer ghp_xxxxxxxxxxxxxx" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/orgs/<Org>/team/<Team Slug>/copilot/usage
[

]
